cli/command/new/templates.go

```diff
@@ -97,7 +98,7 @@ func FetchAndExtractTemplate(org string, repo string) error {
 		return err
 	}
 	if artifactStream == nil {
-		return errors.Errorf("invalid download stream for tarball")
+		return errors.New("invalid download stream for tarball")
```

Comment:
@FriggaHel Could you pls help to confirm whether it's fine?

Reply:
Yup it's fine. The formatting capability is not used here